src/types/environment.d.ts

```diff
@@ -8,6 +8,7 @@ declare global {
 			PORT: string;
 			NPM_CONFIG_LOGLEVEL: string;
 			LOG_LEVEL: string | 'info';
+			LOCAL_STORE_TTL: string | undefined;
```

Comment:
Don't we need this set in the server / container config as well?

You're overriding during runtime, but ideally we want this explicitly set.